### PR TITLE
TPV Initial Project Setup

### PR DESCRIPTION
**Pull request by PureWeen**
### Description of Change ###

TPV Initial Project Setup

### CODE REVIEW

File: Xamarin.Forms.ControlGallery.WindowsUniversal/Xamarin.Forms.ControlGallery.WindowsUniversal.csproj [hunk 1]
```diff
@@ -82,6 +82,7 @@
     <UseVSHostingProcess>false</UseVSHostingProcess>
     <ErrorReport>prompt</ErrorReport>
     <Prefer32Bit>true</Prefer32Bit>
+    <UseDotNetNativeToolchain>false</UseDotNetNativeToolchain>
```
Comment (PureWeen) [inline Xamarin.Forms.ControlGallery.WindowsUniversal/Xamarin.Forms.ControlGallery.WindowsUniversal.csproj]:
```suggestion
```

File: Xamarin.Forms.Platform.UAP/Xamarin.Forms.Platform.UAP.csproj [hunk 2]
```diff
@@ -1,7 +1,4 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras">
-  <PropertyGroup Condition="$(TargetFramework) == 'uap10.0.14393'">
```
Comment (samhouts) [inline Xamarin.Forms.Platform.UAP/Xamarin.Forms.Platform.UAP.csproj]: Intentional?

File: Xamarin.Forms.DualScreen/TwoPaneViewLayoutGuide.shared.cs [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+
+        IDualScreenService DualScreenService =>
+            DependencyService.Get<IDualScreenService>() ?? NoDualScreenServiceImpl.Instance;
+
+        Rectangle _hinge;
+        Rectangle _leftPage;
+        Rectangle _rightPane;
+        TwoPaneViewMode _mode;
+        Layout _layout;
+        private bool isLandscape;
```
Comment (samhouts) [inline Xamarin.Forms.DualScreen/TwoPaneViewLayoutGuide.shared.cs]:
```suggestion
        bool _isLandscape;
```

File: Xamarin.Forms.DualScreen/TwoPaneViewLayoutGuide.shared.cs [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+        public event PropertyChangedEventHandler PropertyChanged;
+        List<string> _pendingPropertyChanges = new List<string>();
+
+        private TwoPaneViewLayoutGuide()
```
Comment (samhouts) [inline Xamarin.Forms.DualScreen/TwoPaneViewLayoutGuide.shared.cs]:
```suggestion
        TwoPaneViewLayoutGuide()
```

File: Xamarin.Forms.DualScreen/TwoPaneViewLayoutGuide.shared.cs [hunk 5]
```diff
@@ -0,0 +1,241 @@
...
+            if (args.PropertyName == nameof(Device.Info.CurrentOrientation))
+            {
+                UpdateLayouts();
+            }
+        }
+
+        public bool IsLandscape
+        {
+            get => DualScreenService.IsLandscape;
+            set => SetProperty(ref isLandscape, value);
```
Comment (samhouts) [inline Xamarin.Forms.DualScreen/TwoPaneViewLayoutGuide.shared.cs]:
```suggestion
            set => SetProperty(ref _isLandscape, value);
```

File: Xamarin.Forms.Platform.UAP/Xamarin.Forms.Platform.UAP.csproj (hunk 2 above)
Comment (PureWeen) [reply Xamarin.Forms.Platform.UAP/Xamarin.Forms.Platform.UAP.csproj]: Yea there's already code that sets this constant here

https://github.com/xamarin/Xamarin.Forms/blob/master/UWP.Build.props#L9

**Pull request merged**